### PR TITLE
[RFR] Minimax in progress

### PR DESCRIPTION
**Pull request by JulienMattiussi**
https://trello.com/c/vpPD3Nkv

Testing routines and minimax concepts
Include a minimax service poc which remains unused for now
Fixing calls of API, it must never bug in return for now

### CODE REVIEW

**Comment by Kmaschta** [comment]
Needs rebase

File: src/tests/ai_minimax_test.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package tests
+
+import (
+	"fmt"
+	"quarto/ai"
+	"quarto/state"
+	"testing"
+)
+
+func estInitAllTreeShouldReturnTree(t *testing.T) {
```
Comment (Luwangel) [inline src/tests/ai_minimax_test.go]: Change `estInitAllTreeShouldReturnTree` to `TestInitAllTreeShouldReturnTree`

**Review by Luwangel — CHANGES_REQUESTED** [review]

File: src/tests/ai_minimax_test.go (hunk 1 above)
Comment (Kmaschta) [reply src/tests/ai_minimax_test.go]: s/estInitAllTreeShouldReturnTree/TestInitAllTreeShouldReturnTree/ ?

Comment (Kmaschta) [reply src/tests/ai_minimax_test.go]: Sedy is installed by default on all marmelab repositories :trophy:

**Review by Luwangel — APPROVED** [review]

**Pull request merged**